packages/perseus/src/widgets/__tests__/radio/focus-ring.test.tsx

```diff
@@ -1,3 +1,4 @@
+import {describe, it} from "@jest/globals";
```

Comment:
I actually don't mind needing to import these Jest symbols.  In my mind, having them implicitly available globally always feels a bit weird.